pysat/instruments/pysat_testing.py

```diff
@@ -23,21 +23,17 @@
 # dictionary of data 'tags' and corresponding description
 # tags are used to choose the behaviour of dummy1
 tags = {'': 'Regular testing data set',
-        'ascend': 'Ascending Integers from 0 testing data set',
-        'descend': 'Descending Integers from 0 testing data set',
-        'plus10': 'Ascending Integers from 10 testing data set',
-        'fives': 'All 5s testing data set',
-        'mlt_offset': 'dummy1 is offset by five from regular testing set',
```

Comment:
These tags were left over from the removed constellation function.  I've pulled them out (as well as Lines 221-232) to clean up our coverage since these are unused.  I think that if we want to add in that functionality in a new subpackage as discussed, we should use `custom_attach` to modify the test data rather than have a bunch of specialized tags here.